qunit-dom/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "qunit-dom",
+  "version": "2.0.0",
+  "description": "High Level DOM Assertions for QUnit",
+  "keywords": [
+    "browser",
+    "dom",
+    "ember-addon",
+    "html",
+    "qunit",
+    "qunit-plugin",
+    "testing"
+  ],
+  "repository": "https://github.com/simplabs/qunit-dom",
+  "license": "MIT",
+  "author": "simplabs GmbH",
+  "types": "dist/qunit-dom.d.ts",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "changelog": "lerna-changelog",
+    "lint": "eslint . --cache",
+    "lint:fix": "eslint . --fix",
+    "prepublish": "rollup -c",
+    "release": "release-it",
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "test:watch": "jest --watchAll --notify"
+  },
+  "jest": {
+    "testEnvironment": "jsdom",
+    "preset": "ts-jest",
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/dist/"
+    ]
+  },
+  "dependencies": {
+    "broccoli-funnel": "^3.0.8",
+    "broccoli-merge-trees": "^4.2.0",
+    "ember-cli-babel": "^7.26.10",
+    "ember-cli-version-checker": "^5.1.2"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "2.0.0",
+    "@ember/test-helpers": "2.9.4",
```

Comment:
Seems like the `qunit-dom` addon code has too many dependencies specified here.
This directory is no longer a host to the ember testing app, so basically all ember deps should be removed.

Reply:
These can be removed once v2 addon conversation happens